internal/terraform/context_apply_deferred_test.go

```diff
@@ -2345,6 +2403,15 @@ func (provider *deferredActionsProvider) Provider() providers.Interface {
 			}
 		},
 		ImportResourceStateFn: func(request providers.ImportResourceStateRequest) providers.ImportResourceStateResponse {
+			if request.ID == "deferred" {
+				return providers.ImportResourceStateResponse{
+					ImportedResources: []providers.ImportedResource{},
+					Deferred: &providers.Deferred{
+						Reason: providers.DeferredReasonAbsentPrereq,
```

Comment:
nit: make this something we expect back from the provider, eg. providers.PROVIDER_CONFIG_UNKNOWN, otherwise it looks like the Terraform system has produced this which isn't the case.